playbooks/roles/course_discovery/defaults/main.yml

```diff
@@ -0,0 +1,154 @@
+---
+#
+# edX Configuration
+#
+# github:     https://github.com/edx/configuration
+# wiki:       https://github.com/edx/configuration/wiki
+# code style: https://github.com/edx/configuration/wiki/Ansible-Coding-Conventions
+# license:    https://github.com/edx/configuration/blob/master/LICENSE.TXT
+#
+##
+# Defaults for role course_discovery
+# 
+COURSE_DISCOVERY_GIT_IDENTITY: !!null
+
+# depends upon Newrelic being enabled via COMMON_ENABLE_NEWRELIC
+# and a key being provided via NEWRELIC_LICENSE_KEY
+COURSE_DISCOVERY_NEWRELIC_APPNAME: "{{ COMMON_ENVIRONMENT }}-{{ COMMON_DEPLOYMENT }}-{{ course_discovery_service_name }}"
+COURSE_DISCOVERY_PIP_EXTRA_ARGS: "-i {{ COMMON_PYPI_MIRROR_URL }}"
+COURSE_DISCOVERY_NGINX_PORT: 18381
+COURSE_DISCOVERY_SSL_NGINX_PORT: 48381
+
+COURSE_DISCOVERY_DEFAULT_DB_NAME: 'course_discovery'
+COURSE_DISCOVERY_MYSQL: 'localhost'
+# MySQL usernames are limited to 16 characters
+COURSE_DISCOVERY_MYSQL_USER: 'course_discov001'
+COURSE_DISCOVERY_MYSQL_PASSWORD: 'password'
```

Comment:
@johnbaldwin Anything in a role's `default/main.yml` can be overridden. All credentials should be overridden in server vars / secrets files.